### ISSUE TITLE
Coloring logs?

### ISSUE THREAD

**Issue by STaRDoGG** [amir20/dozzle#997]
**Is your feature request related to a problem? Please describe.**

Are there already any plans to include coloring of logs? Perhaps using RegEx?

example; a lot of containers just spit out massive amounts of text, like elasticsearch for example::

![image](https://user-images.githubusercontent.com/1524526/107361270-bb045d80-6a9c-11eb-9afe-541e35b6f414.png)

**Describe the solution you'd like**

The ability to set coloring to be able to quickly glean with the eye, errors for example, if they were colored red, would make parsing logs a lot easier.

example: we could color "error" matches with red; "info" messages with yellow, "warn" orange, and so forth.

**Describe alternatives you've considered**

None, just throwing helpful ideas atcha once again, lol.

**Additional context**

Can't recall if we've already talked about this briefly, or if you already have it planned, so I'm just tossing it out there just in case.

**Comment by amir20** [comment]
I don't plan to do this. Someone asked for it and I did some research. It was a lot of work to create a UI that would configure such colors. 

I think the right thing would be to support JSON. I worked on it a little. With JSON support, then it could be configured with colors, filter and hide properties. 

It's a lot of work for JSON support though so I don't think it will happen with my limited time right now.
